<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deformed</class>
 <widget class="QDialog" name="Deformed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1872</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deformed</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="gui::canvas::Deformed" name="canvas" native="true">
     <property name="contextMenuPolicy">
      <enum>Qt::PreventContextMenu</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="panel">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="17">
       <widget class="QLabel" name="label_step">
        <property name="text">
         <string>Step: 0000/0000</string>
        </property>
       </widget>
      </item>
      <item row="1" column="23">
       <widget class="QLabel" name="label_node">
        <property name="text">
         <string>Node: 0000</string>
        </property>
       </widget>
      </item>
      <item row="1" column="19">
       <widget class="QLabel" name="label_time">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Load: +0.00e+00</string>
        </property>
       </widget>
      </item>
      <item row="1" column="29">
       <spacer name="spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="21">
       <widget class="QLabel" name="label_time_max">
        <property name="text">
         <string>Max: +0.00e+00</string>
        </property>
       </widget>
      </item>
      <item row="1" column="15">
       <widget class="QPushButton" name="button_skip_sub">
        <property name="toolTip">
         <string>Decrement skip</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="list-remove">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="button_first">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>First</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="media-skip-backward">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="button_previous">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Previous</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="media-seek-backward">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="button_pause">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Pause</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="media-playback-pause">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="button_play">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Play</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="media-playback-start">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="button_next">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Next</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="media-seek-forward">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="button_stop">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Stop</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="media-playback-stop">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QPushButton" name="button_last">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Last</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="media-skip-forward">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QPushButton" name="button_print">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Print</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="document-print">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QPushButton" name="button_record">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Record</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="media-record">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="QPushButton" name="button_plot">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Options</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="document-properties">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="12">
       <widget class="QLabel" name="label_skip">
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Skip:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="13">
       <widget class="QPushButton" name="button_skip_add">
        <property name="toolTip">
         <string>Increment skip</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="list-add">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="14">
       <widget class="QLineEdit" name="edit_skip">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>001</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="25">
       <widget class="QLabel" name="label_dof">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Dof: +0.00e+00</string>
        </property>
       </widget>
      </item>
      <item row="1" column="28">
       <widget class="Line" name="line_end">
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="20">
       <widget class="QLabel" name="label_time_min">
        <property name="text">
         <string>Min: +0.00e+00</string>
        </property>
       </widget>
      </item>
      <item row="1" column="16">
       <widget class="Line" name="line_step">
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="22">
       <widget class="Line" name="line_dof">
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="18">
       <widget class="Line" name="line_load">
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="26">
       <widget class="QLabel" name="label_dof_min">
        <property name="text">
         <string>Min: +0.00e+00</string>
        </property>
       </widget>
      </item>
      <item row="1" column="27">
       <widget class="QLabel" name="label_dof_max">
        <property name="text">
         <string>Max: +0.00e+00</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="30">
       <widget class="QSlider" name="slider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="24">
       <widget class="QLabel" name="label_type">
        <property name="text">
         <string>Type: ux</string>
        </property>
       </widget>
      </item>
      <item row="1" column="10">
       <widget class="QPushButton" name="button_plots">
        <property name="toolTip">
         <string>Plots</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="go-home">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="11">
       <widget class="QPushButton" name="button_paths">
        <property name="toolTip">
         <string>Paths</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="applications-graphics">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="16">
       <widget class="QLabel" name="label_header_controls">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Controls</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="17">
       <widget class="QLabel" name="label_header_steps">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Steps</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="19" colspan="3">
       <widget class="QLabel" name="label_header_solver">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Solver</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="23" colspan="5">
       <widget class="QLabel" name="label_header_dof">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Watch DOF</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>gui::canvas::Deformed</class>
   <extends>QWidget</extends>
   <header location="global">Canvas/DeformedCanvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
